Makefile.third

```diff
@@ -495,3 +495,11 @@ $(EVERNOTE_LIB): $(THIRDPARTY_DIR)/evernote-sdk-lua/CMakeLists.txt
 $(LUALONGNUMBER): $(EVERNOTE_LIB) $(THIRDPARTY_DIR)/evernote-sdk-lua/CMakeLists.txt
 	cp $(CURDIR)/$(EVERNOTE_PLUGIN_DIR)/lib/liblualongnumber.so \
 		$(CURDIR)/$(OUTPUT_DIR)/libs
+
+$(SQLITE_LIB): $(THIRDPARTY_DIR)/sqlite/CMakeLists.txt
+	install -d $(SQLITE_BUILD_DIR)
+	cd $(SQLITE_BUILD_DIR) && \
+		$(CMAKE) -DCC="$(CC)" \
+			-DCFLAGS="$(if $(EMULATE_READER),,-fno-ffast-math)" \
```

Comment:
I made a typo in my original post, the actual name of the flag is ```-fno-fast-math``` (and we need it always, there's an #error preprocessor define if fast-math is enabled).